toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -532,23 +531,27 @@ func validateScripts(baseConfigPath string, scripts *imagecustomizerapi.Scripts)
 }
 
 func validateScript(baseConfigPath string, script *imagecustomizerapi.Script) error {
-	// Ensure that install scripts sit under the config file's parent directory.
-	// This allows the install script to be run in the chroot environment by bind mounting the config directory.
-	if !filepath.IsLocal(script.Path) {
-		return fmt.Errorf("install script (%s) is not under config directory (%s)", script.Path, baseConfigPath)
-	}
+	if script.Path != "" {
+		// Ensure that install scripts sit under the config file's parent directory.
+		// This allows the install script to be run in the chroot environment by bind mounting the config directory.
+		if !filepath.IsLocal(script.Path) {
+			return fmt.Errorf("script (%s) is not under config directory (%s)", script.Path, baseConfigPath)
+		}
 
-	// Verify that the file exists.
-	fullPath := filepath.Join(baseConfigPath, script.Path)
+		// Verify that the file exists.
+		fullPath := filepath.Join(baseConfigPath, script.Path)
 
-	scriptStat, err := os.Stat(fullPath)
-	if err != nil {
-		return fmt.Errorf("couldn't read install script (%s):\n%w", script.Path, err)
-	}
+		if script.Interpreter == "" {
+			scriptStat, err := os.Stat(fullPath)
```

Comment:
Should this check apply whether an interpreter is specified or not?

Reply:
No. The execute bit isn't required if you are explicitly specifying the interpreter program, as opposed to directly executing the script.

Reply:
I meant the check to see if the file exists or not should be independent of whether the interpreter is specified or not. I see that in the latest iteration you are always checkng for the presence of the file independent of interpreter value.